packages/wonder-blocks-clickable/src/util/is-client-side-url.js

```diff
@@ -2,12 +2,13 @@
 /**
  * Returns:
  * - false for hrefs staring with http://, https://, //.
- * - false for '#' and 'javascript:void(0);'
+ * - false for '#', 'javascript:...', 'mailto:...', 'tel:...', etc.
  * - true for all other values, e.g. /foo/bar
  */
 export const isClientSideUrl = (href: string): boolean => {
     return (
         !/^(https?:)?\/\//i.test(href) &&
-        !/^(#|javascript:void\(0\);?)$/.test(href)
+        !/^[a-zA-Z]+:/.test(href) &&
+        href !== "#"
```

Comment:
I'm not sure if we have any case where we define a hash (e.g. `href="#math"`), would we need to cover that scenario as well?

Reply:
We don't do that for any `Link`, but we'll probably swap out some of our `<a>`s for `<Link>`s in the future so we may as well be ready for that.